integreat_cms/static/src/js/pois/poi-map.ts

```diff
@@ -0,0 +1,51 @@
+import maplibregl from "maplibre-gl";
+
+window.addEventListener("load", () => {
+  let longitude = document.getElementById("id_longitude") as HTMLInputElement;
+  let latitude = document.getElementById("id_latitude") as HTMLInputElement;
+
+  let map = document.getElementById("map");
+
+  showPreview(map);
+
+  longitude.addEventListener("focusout", ()=> showPreview(map));
+  latitude.addEventListener("focusout",  ()=> showPreview(map));
+
+  longitude.addEventListener("input", ()=> showPreview(map));
+  latitude.addEventListener("input", ()=> showPreview(map));
+
+  document.getElementById("auto-fill-coordinates")?.addEventListener("input", ({ target }) => {
+    if (!(target as HTMLInputElement).checked) {
+      showPreview(map);
+    }
+  });
```

Comment:
```suggestion
```
I'm not sure, do we really need this event listener? Probably it's sufficient that enabling the option causes the reload of the coordinates which dispatches the input event and already updates the map...